pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerServiceTest.java

```diff
@@ -1666,4 +1666,29 @@ public void testIsSystemTopicAllowAutoTopicCreationAsync() throws Exception {
         assertTrue(brokerService.isAllowAutoTopicCreationAsync(
                 "persistent://pulsar/system/my-system-topic").get());
     }
+
+    @Test
+    public void testDuplicateAcknowledgement() throws Exception {
+        final String ns = "prop/ns-test";
+
+        admin.namespaces().createNamespace(ns, 2);
+        final String topicName = "persistent://prop/ns-test/duplicated-acknowledgement-test";
+        @Cleanup
+        Producer<byte[]> producer = pulsarClient.newProducer()
+                .topic(topicName)
+                .create();
+        @Cleanup
+        Consumer<byte[]> consumer1 = pulsarClient.newConsumer()
+                .topic(topicName)
+                .subscriptionName("sub-1")
```

Comment:
```
                .isAckReceiptEnabled(true)
```
add this config, otherwise, it will lead to test instability.

Reply:
fixed